src/Approvals/SFA.DAS.Approvals/Application/Cohorts/Queries/GetConfirmEmployerDeclaredStandards/GetConfirmEmployerDeclaredStandardsQueryHandler.cs

```diff
@@ -2,23 +2,23 @@
 using System.Threading;
 using System.Threading.Tasks;
 using MediatR;
+using Microsoft.IdentityModel.Tokens;
 using SFA.DAS.Approvals.InnerApi.Requests;
 using SFA.DAS.Approvals.InnerApi.Responses;
 using SFA.DAS.Approvals.Services;
 using SFA.DAS.SharedOuterApi.Configuration;
 using SFA.DAS.SharedOuterApi.Extensions;
 using SFA.DAS.SharedOuterApi.Interfaces;
-using Party = SFA.DAS.Approvals.Application.Shared.Enums.Party;
 
-namespace SFA.DAS.Approvals.Application.Cohorts.Queries.GetHasDeclaredStandards
+namespace SFA.DAS.Approvals.Application.Cohorts.Queries.GetConfirmEmployerDeclaredStandards
 {
-    public class GetHasDeclaredStandardsQueryHandler : IRequestHandler<GetHasDeclaredStandardsQuery, GetHasDeclaredStandardsQueryResult>
+    public class GetConfirmEmployerDeclaredStandardsQueryHandler : IRequestHandler<GetConfirmEmployerDeclaredStandardsQuery, GetConfirmEmployerDeclaredStandardsQueryResult>
     {
         private readonly ICommitmentsV2ApiClient<CommitmentsV2ApiConfiguration> _apiClient;
         private readonly IProviderStandardsService _providerStandardsService;
         private readonly ServiceParameters _serviceParameters;
 
-        public GetHasDeclaredStandardsQueryHandler(
+        public GetConfirmEmployerDeclaredStandardsQueryHandler(
```

Comment:
Sorry if I wasn't clear before - I just think this should be called `GetConfirmEmployerQueryHandler` eg. without any reference to DeclaredStandards in the name. This aligns it with the approach for the other pages.

Reply:
Done